### PR TITLE
Added the ability to send an email (once a day) for bad credentials

### PR DESCRIPTION
**Pull request by neybar**
I wanted the ability to get a notification if my credentials expire.  

I decided to hard code in the requirement to only send once a day, but that could be moved to a config option pretty easy.

I also waffled on if I wanted to make the getters for the smtp variables chatty like the rest are.  I ended up making them behave similar to how you already have it. 

Let me know if you have any feedback.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -10,6 +10,14 @@ settings:
   destination: './drive'
   # verbose
   verbose: false
+smtp:
+  # If you want to recieve email notifications about expired/missing 2FA credentials then uncommend
```
Comment (mandarons) [inline config.yaml]: Typo in 'uncommend'

File: src/notify.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        try:
+            sent_on = datetime.datetime.now()
+            if not dry_run:
+                smtp = smtplib.SMTP(host, port)
+                smtp.set_debuglevel(0)
+                smtp.connect(host, port)
+                if not no_tls:
+                    smtp.starttls()
+
+                if password:
```
Comment (mandarons) [inline src/notify.py:30]: This check isn't required as it is already done on line 19

**Review by mandarons — CHANGES_REQUESTED** [review]
The feature looks great as a first iteration! Added few comments. Also, 100% test coverage is required.

**Comment by mandarons** [comment]
Thank you, @neybar, for submitting the first PR to this project! I appreciate you taking some time to improve this project. 🥇 

I like your feature suggestion. I see that there are few things we can bring into the config file - such as frequency of email, To email etc. But let's leave it for some other day. 😄

File: config.yaml (hunk 1 above)
Comment (neybar) [reply config.yaml]: fixed

File: src/notify.py (hunk 2 above)
Comment (neybar) [reply src/notify.py:30]: Updated.  I pulled password check out of line 19

**Review by mandarons — APPROVED** [review]
Looks good.

**Pull request merged**